frontend-react/src/components/ReportStreamFooter.tsx

```diff
@@ -63,19 +68,19 @@ export const ReportStreamFooter = () => {
                     </IdentifierLink>
                 </IdentifierLinkItem>
                 <IdentifierLinkItem>
-                    <IdentifierLink href="https://oig.hhs.gov/">
+                    <USExtLink href="https://oig.hhs.gov/">
```

Comment:
I get the requirement for the external icon launcher thing, but is it okay that these links are differently colored?  Or is that intended?

<img width="941" alt="Screenshot 2023-02-08 at 21 33 29" src="https://user-images.githubusercontent.com/2421042/217702890-2ada3340-dd18-4b02-a102-00f547cc39db.png">

Reply:
Yes, have the same question as Stephen about colors. Should be the same colors as the others if possible.